packages/plugin-analytic-fb-pixel/src/FacebookPixelAnalytic.ts

```diff
@@ -1,59 +1,71 @@
-import { AbstractAnalytic, defaultDependencies } from '@ima/plugin-analytic';
+import { Dependencies } from '@ima/core';
+import { AbstractAnalytic } from '@ima/plugin-analytic';
 
 const FB_ROOT_VARIABLE = 'fbq';
 
+export type AnalyticFBPixelSettings = {
+  id: string | null;
+};
+
 /**
  * Facebook Pixel Helper.
  *
  * @class
  */
-export default class FacebookPixelAnalytic extends AbstractAnalytic {
-  /** @type {import('@ima/core').Dependencies} */
-  static get $dependencies() {
-    return [...defaultDependencies, '$Settings.plugin.analytic.fbPixel'];
+export class FacebookPixelAnalytic extends AbstractAnalytic {
+  #config: AnalyticFBPixelSettings;
+  // An identifier for Facebook Pixel.
+  _id: string | null;
```

Comment:
podtrzitka ted pouzivame pro protected veci, bude se tahle trida rozsirovat? respektive budou potreba tyhle properties jako protected? to same plati i pro metody, udelal bych nejaky preuzkum, jestli nejde neco schovat jeste do private primo..procisti to pak ten interface trochu (i pro jiny soubory v jinem repu.. :D)

Reply:
prošel jsem a upravil co mi dávalo smysl.. konrkétně `_id` se nikde nepoužívalo, takže jsem smazal :D